tsconfig.json

```diff
@@ -22,7 +22,8 @@
     "jsx": "preserve",
     "downlevelIteration": true,
     "allowSyntheticDefaultImports": true,
-    "types": ["react-spring", "jest"]
+    "types": ["react-spring", "jest"],
+    "baseUrl": "src"
```

Comment:
you sure you don't need to setup `paths` like we do in **dex-react**
I have a suspicion if you had different types in your `Header` TS would still resolve from old location

Reply:
I did it at first, but then I tried without, and it worked. So I prefered to make the minimum changes

Reply:
Ctrl+click `Header` inside `App.tsx`, see where it leads. That's where TS resolves it form